diff --git a/third_party/lottie/minify_lottie.py b/third_party/lottie/minify_lottie.py
--- a/third_party/lottie/minify_lottie.py
+++ b/third_party/lottie/minify_lottie.py
@@ -12,16 +12,14 @@ _SRC_PATH = os.path.normpath(os.path.join(_HERE_PATH, '..', '..'))
 
 sys.path.append(os.path.join(_SRC_PATH, 'third_party', 'node'))
 import node
-import node_modules
 
 def main(argv):
   parser = argparse.ArgumentParser()
   parser.add_argument('--out_folder', required=True)
   args = parser.parse_args(argv)
 
-  node.RunNode([node_modules.PathToTerser(),
+  node.RunNode(['/usr/bin/node',
           os.path.join(_HERE_PATH, 'lottie_worker.js'),
-          '--ascii_only=true',
           '-b', 'beautify=false',
           '--compress',
           '--mangle', 'reserved=[\'$\',\'onmessage\',\'postMessage\']',
diff --git a/third_party/node/node.gni b/third_party/node/node.gni
--- a/third_party/node/node.gni
+++ b/third_party/node/node.gni
@@ -22,9 +22,6 @@ template("node") {
 
     # When use_remoteexec=true, node actions may run on remote
     # Linux worker. So it should include linux node binary in inputs.
-    if (is_linux || is_chromeos || use_remoteexec) {
-      inputs += [ "//third_party/node/linux/node-linux-x64/bin/node" ]
-    }
     if (is_win && host_os == "win") {
       inputs += [ "//third_party/node/win/node.exe" ]
     }
diff --git a/third_party/node/node.py b/third_party/node/node.py
--- a/third_party/node/node.py
+++ b/third_party/node/node.py
@@ -11,17 +11,7 @@ import os
 
 
 def GetBinaryPath():
-  if platform.machine() == 'arm64':
-    darwin_path = 'mac_arm64'
-    darwin_name = 'node-darwin-arm64'
-  else:
-    darwin_path = 'mac'
-    darwin_name = 'node-darwin-x64'
-  return os_path.join(os_path.dirname(__file__), *{
-    'Darwin': (darwin_path, darwin_name, 'bin', 'node'),
-    'Linux': ('linux', 'node-linux-x64', 'bin', 'node'),
-    'Windows': ('win', 'node.exe'),
-  }[platform.system()])
+  return '/usr/bin/node'
 
 
 def RunNode(cmd_parts, stdout=None):
diff --git a/tools/typescript/tsconfig_base.json b/tools/typescript/tsconfig_base.json
--- a/tools/typescript/tsconfig_base.json
+++ b/tools/typescript/tsconfig_base.json
@@ -3,7 +3,7 @@
     "module": "ESNext",
     "target": "ESNext",
 
-    "noEmitOnError": true,
+    "noEmitOnError": false,
     "pretty": true,
 
     "allowUnreachableCode": false,
diff --git a/third_party/devtools-frontend/src/scripts/devtools_paths.py b/third_party/devtools-frontend/src/scripts/devtools_paths.py
--- a/third_party/devtools-frontend/src/scripts/devtools_paths.py
+++ b/third_party/devtools-frontend/src/scripts/devtools_paths.py
@@ -38,13 +38,6 @@ def third_party_path():

 # This points to the node binary downloaded as part of the checkout.
 def node_path():
-    try:
-        old_sys_path = sys.path[:]
-        sys.path.append(path.join(third_party_path(), 'node'))
-        import node
-    finally:
-        sys.path = old_sys_path
-    return node.GetBinaryPath()
-
+    return '/usr/bin/node'

 def devtools_root_path():
diff --git a/third_party/protobuf/proto_library.gni b/third_party/protobuf/proto_library.gni
--- a/third_party/protobuf/proto_library.gni
+++ b/third_party/protobuf/proto_library.gni
@@ -260,10 +260,6 @@ _protoc_gen_ts_runtime_deps = [
 
 # When use_remoteexec=true, node actions may run on remote
 # Linux worker. So it should include linux node binary in inputs.
-if (is_linux || is_chromeos || use_remoteexec) {
-  _protoc_gen_ts_runtime_deps +=
-      [ "//third_party/node/linux/node-linux-x64/bin/node" ]
-}
 if (is_win && host_os == "win") {
   _protoc_gen_ts_runtime_deps += [ "//third_party/node/win/node.exe" ]
 }
